assets/js/components/sell-quick-start.component.js

```diff
@@ -144,6 +150,8 @@ function sellQuickStartController ($scope, $rootScope, currency, buySell, Alerts
       if (r.absoluteFeeBounds[0] === 0) {
         $scope.error['moreThanInWallet'] = true;
         $scope.offerUseAll();
+      } else {
+        $scope.status = {};
```

Comment:
Ok, makes sense. Agree that there's probably a more efficient way to build the payment, but might not be critical right now. Thanks 👍